Engine/platform/android/acpland.cpp

```diff
@@ -611,40 +610,40 @@ bool ReadConfiguration(char* filename, bool read_everything)
 
 //    ReadInteger((int*)&psp_return_to_menu, "misc", "return_to_menu", 0, 1, 1);
 
-    ReadString(&psp_translation[0], cfg, "misc", "translation", "default");
+    ReadString(&psp_translation[0], cfg, (char*) "misc", (char*) "translation", (char*) "default");
 
-    ReadInteger((int*)&psp_config_enabled, cfg, "misc", "config_enabled", 0, 1, 0);
+    ReadInteger((int*)&psp_config_enabled, cfg, (char*) "misc", (char*) "config_enabled", 0, 1, 0);
```

Comment:
I noticed these changes in previous variants, probably dating back to sonneveld's work, and hoped this won't be in pr.

There's no reason for ReadInteger and similar functions to demand char* pointer, as it is not supposed to change the string. The correct way is instead to fix the ReadInteger and other function and change their "char*" arguments to "const char*".

Reply:
Right, I will look into doing it this way.

Reply:
I believe I updated the right places! Can you check if it's correct?

Reply:
Yes it's fine now.